latest/ug/cluster-management/cost-monitoring-kubecost.adoc

```diff
@@ -45,7 +45,7 @@ Learn how to <<removing-an-add-on,remove an EKS Add-on>>, such as Kubecost.
 
 * An existing Amazon EKS cluster. To deploy one, see <<getting-started>>.
 * The `kubectl` command line tool is installed on your device or {aws} CloudShell. The version can be the same as or up to one minor version earlier or later than the Kubernetes version of your cluster. For example, if your cluster version is `1.29`, you can use `kubectl` version `1.28`, `1.29`, or `1.30` with it. To install or upgrade `kubectl`, see <<install-kubectl>>.
-* Helm version 3.9.0 or later configured on your device or {aws} CloudShell. To install or update Helm, see <<helm>>.
+* https://helm.sh/docs/topics/version_skew/#supported-version-skew[Supported Helm version] on your device or {aws} CloudShell. To install or update Helm, see <<helm>>.
```

Comment:
Kubernetes version changes from time to time, and 3.9.0 only support up to Kubernetes 1.21 version.